lib/jira/plugin.rb

```diff
@@ -49,7 +52,7 @@ def check(key: nil, url: nil, emoji: ":link:", search_title: true, search_commit
         search_commits: search_commits
       )
 
-      if !jira_issues.empty?
+      if !jira_issues.empty? && !messages_on_non_issues
```

Comment:
Suggestion: What are your thoughts on changing this to something like `skip_on_no_issues`? You are already indicating the word skip in the param comments above.

Reply:
yeah that works better, will add to readme as well.